assets/parquet/main.py

```
@@ -0,0 +1,154 @@
+            ); \
+            """)
+            self.docker_execute([
+                f"python3 {ASSETS_DIR}/ingest_json_string.py {self.datasets_path}"
+                ])
+
+    
+    def search(self, query):
+        """
+        Searches an already-ingested dataset with query, which is populated within config.yaml
+        """
+        res = self.hive_execute(f"SELECT * from {PARQUET_TABLE_NAME} WHERE {query.strip()[1:-1]}").strip()
```

Ruff (0.12.2)

Possible SQL injection vector through string-based query construction

(S608)